oqum/risk/00_workflow_event_based_risk.tex

```diff
@@ -0,0 +1,64 @@
+This calculator employs an event-based Monte Carlo simulation approach to
+probabilistic risk assessment in order to estimate the loss distribution for
+individual assets and aggregated loss distribution for a spatially distributed
+portfolio of assets within a specified time period. The calculator requires
+the definition of an exposure model, a vulnerability model for each loss type
+of interest with vulnerability functions for each taxonomy represented in the
+exposure model, and a set of ground motion fields representative of the
+seismicity of the region over the specified time period. Loss curves and loss
+maps can currently be calculated for five different loss types using this
+calculator: structural losses, nonstructural losses, contents losses, downtime
+losses, and occupant fatalities. The main results of this calculator are loss
+exceedance curves for each asset, which describe the probability of exceedance
+of different loss levels over the specified time period, and loss maps for the
+region, which describe the loss values that have a given probability of
+exceedance over the specified time period. Aggregate loss exceedance curves
+can be also be produced using this calculator; these describe the probability
+of exceedance of different loss levels for all assets of a single taxonomy, or
+for all assets in the portfolio, over the specified time period. Finally,
```

Comment:
I think we produce the aggregate loss curves only for all assets, not for the assets of a given taxonomy.